larray_editor/editor.py

```diff
@@ -678,8 +678,49 @@ def open_tutorial(self):
     def open_api_documentation(self):
         QDesktopServices.openUrl(QUrl(urls['doc_api']))
 
-    def open_issue_tracker(self):
-        QDesktopServices.openUrl(QUrl(urls['issue_tracker']))
+    def report_issue(self):
+        if PY2:
+            from urllib import quote
+        else:
+            from urllib.parse import quote
+
+        versions = get_versions()
+        issue_template = """\
+## Description
+**What steps will reproduce the problem?**
+1. 
+2. 
+3.
+ 
+**What is the expected output? What do you see instead?**
+
+
+**Please provide any additional information below**
+
+
+## Version and main components
+* Python {python} on {system} {bitness:d}bits
+* Qt {qt}, {qt_api} {qt_api_ver}
+"""
+        if versions.get('larray'):
+            issue_template += "* larray {larray}\n"
```

Comment:
In this template, I wonder if it wouldn't be better to have all libs appear (even if not present) and have their version display as n/a. It would make it more obvious they are missing.